src/ia-account-settings.ts

```diff
@@ -187,6 +187,13 @@ export class IAAccountSettings
    */
   @state() private apiHasExecuted?: Boolean = false;
 
+  /**
+   * Stores the authentication method used (e.g. 'ia' for Internet Archive)
+   * @type {String}
+   * @memberof IAUXAccountSettings
+   */
+  @state() private authBy?: String = '';
```

Comment:
Does this need to be optional since you're providing a default value? 

Also the type should be lowercase `string` instead of `String` since it's a typescript type. Uppercase `String` gets used for `@property` decorators since they're Lit types, ie 
```
@property({ type: String }) foo: string = ''
```

Reply:
considering below suggestion and we don't need to make it property

Reply:
Ah, there are few things I can improve here. Thank you!